libr/anal/anal.c

```diff
@@ -577,6 +577,38 @@ R_API bool r_anal_noreturn_at_addr(RAnal *anal, ut64 addr) {
 	return sdb_bool_get (anal->sdb_types, K_NORET_ADDR(addr), NULL);
 }
 
+bool noreturn_recurse(RAnal *anal, ut64 addr) {
+	RAnalOp op = {0};
+	ut8 bbuf[0x10];
```

Comment:
initialize or check the return value of read_at()

Reply:
"and"

Reply:
What variable should be initialized?

Reply:
Bbuf[0x10] = {0};